src/base/QXmppUtils.cpp

```diff
@@ -280,8 +283,11 @@ int QXmppUtils::generateRandomInteger(int N)
 {
     Q_ASSERT(N > 0 && N <= RAND_MAX);
     int val;
-    while (N <= (val = qrand() / (RAND_MAX / N))) {
-    };
+#if QT_VERSION >= QT_VERSION_CHECK(5, 10, 0)
+    while (N <= (val = QRandomGenerator::global()->generate() / (RAND_MAX / N))) { }
+#else
+    while (N <= (val = qrand() / (RAND_MAX / N))) { }
```

Comment:
Can you run clang-format on this? (`clang-format -i src/base/QXmppUtils.cpp` from repository root)